homeassistant/components/zwave_js/lock.py

```diff
@@ -34,6 +39,9 @@
     },
 }
 
+SERVICE_SET_USERCODE = "set_usercode"
+SERVICE_CLEAR_USERCODE = "clear_usercode"
```

Comment:
We should add lock to the service name. Now it's registered under zwave_js domain but it's not clear it's lock specific.

```suggestion
SERVICE_SET_USERCODE = "set_usercode"
SERVICE_CLEAR_USERCODE = "clear_usercode"
```

@MartinHjelmare do you agree?

Reply:
maybe `set_lock_usercode` and `clear_lock_usercode` then?

Reply:
Yes.